cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/TestCommandTest.java

```diff
@@ -72,6 +72,18 @@ public void testTestBalFile() throws IOException {
         testCommand.execute();
     }
 
+    @Test(description = "Test a valid ballerina file with periods in the file name")
+    public void testTestBalFileWithPeriods() throws IOException {
+        Path validBalFilePath = this.testResources.resolve("valid-test-bal-file").resolve("sample.tests.bal");
+
+        System.setProperty("user.dir", this.testResources.resolve("valid-test-bal-file").toString());
```

Comment:
Can use `ProjectConstants.USER_DIR`

Reply:
Changed all instances of "user.dir" to ProjectConstants.USER_DIR